### ISSUE TITLE
Search : Incorrect URL and page crash on topic links 

### ISSUE THREAD

**Issue by rupalichavan9860** [the-canonizer/Canonizer-Beta--Issue-Tracking#1559]
Steps to reproduce : 
1. Go to any topic detail page 
2. Search any keyword in search
3. Click on canon name 
![image](https://github.com/user-attachments/assets/d3575bf4-0374-4660-9178-bc71604025be)

4. Observe URL 

https://canonizer.com/topic/[...camp] 
5. Open in new tab 

Getting page crash : 
![image](https://github.com/user-attachments/assets/a4dbbe41-0701-4362-9e42-c96741eef45a)

**Comment by shavetatalentelgia** [comment]
@saniatg , @rupalichavan9860 , Please re-verify this ticket. As I am unable to reproduce this issue at my end.

**Comment by rupalichavan9860** [comment]
Yes @shavetatalentelgia It redirects to the search page. Can we remove the hand pointer and keep the usual pointer because it does not redirect to any other page?

**Comment by shavetatalentelgia** [comment]
Yes, Canon is not anchor. I will remove the hand pointer from canon. Only this modification has to be done right. As Page crash is not reproducing at my end.

**Comment by rupalichavan9860** [comment]
> @rupalichavan9860 , Agree, Canon is not anchor. I will remove the hand pointer from canon. Only this modification has to be done right.
> 
> As Page crash is not reproducing at my end.

Yes. The page is not crashing currently.

**Comment by rupalichavan9860** [comment]
Verified on UX server

**Comment by rupalichavan9860** [comment]
Verified on live
